README.md

```diff
@@ -151,6 +152,14 @@ status, input and both outputs to it:
 Cheetah.run("ls -l", logger: logger)
 ```
 
+### Overwritting env
+
+If command need to have own specified ENV set, then passing :env option does it.
+
+```ruby
+  Cheetah.run("env", env: { "LC_ALL" => "C" })
+```
```

Comment:
This should also explain whether the option resets the ENV or updates it. Issue #17 mentions `nil` as unsetting a variable. So?